src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -0,0 +1,31 @@
+import { Person } from '../../types/Person';
+import { DropdownItem } from '../DropdownItem/DropdovnItem';
+
+type Props = {
+  people: Person[]
+  onSelect: (person: Person) => void;
+};
+
+export const DropdownMenu: React.FC<Props> = ({ people, onSelect }) => {
+  return (
+    <div className="dropdown-menu" role="menu">
+      <div className="dropdown-content">
```

Comment:
![image](https://github.com/mate-academy/react_autocomplete/assets/86923089/c72f925c-1c0b-4dc6-99ae-3d1b837fc529)

it means if the input is in focus